python/cudf/tests/test_dataframe.py

```diff
@@ -1001,3 +1003,62 @@ def test_dataframe_shape_empty():
     gdf = DataFrame()
 
     assert pdf.shape == gdf.shape
+
+
+@pytest.fixture
+def df():
+    return pd.DataFrame({'x': range(10),
+                         'y': range(10)})
```

Comment:
In the existing tests `pdf` is used really consistently for pandas dataframes.

Reply:
Fixed.  I've also changed the gf pronoun to gdf to match (hopefully this is welcome as well)